nss-ice-phantoms/BangazonSite
# Ticket 5

# Description
Completes Ticket #5 

## Type of change

# Testing Instructions

1. `git fetch --all`
2. `git checkout AP-PCategory`
3. Run Bangazon
4. Verify that you can select product types through a dropdown when creating a new product
5.  verify that you cant add a product unless a product type is selected


# Checklist:
